arches_resource_sets/views/resource_set_member_view.py

```diff
@@ -0,0 +1,58 @@
+from json import JSONDecodeError
+import logging
+from django.db import IntegrityError
+from arches.app.utils.betterJSONSerializer import JSONDeserializer
+from arches.app.views.api import APIBase
+from arches_resource_sets.models import ResourceSet, ResourceSetMember
+from arches.app.utils.response import JSONResponse, JSONErrorResponse
+
+logger = logging.getLogger(__name__)
+class ResourceSetMemberView(APIBase):
+    def get(self, request, set_id):
+        """Returns either a flat representation (?flat=true) or a tree (default)."""
+        try:
+            set_members = ResourceSet.objects.get(pk=set_id).resourcesetmember_set.all().values_list("resource_instance_id", flat=True)
+        except ResourceSet.DoesNotExist:
+            return JSONErrorResponse("Resource Set not found", "Resource Set with ID '{}' not found".format(set_id), status=404)
+        
+        return JSONResponse({"resource_set_members": set_members})
```

Comment:
let's call them what they actually are
```suggestion
        return JSONResponse({"resource_instance_ids": set_members})
```